src/quo/components/wallet/account_permissions/schema.cljs

```diff
@@ -0,0 +1,22 @@
+(ns quo.components.wallet.account-permissions.schema
+  (:require [quo.components.wallet.required-tokens.view :as required-tokens]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:account
```

Comment:
Do you see value in extracting `account` to `schema.common` ?

Reply:
Where to put schemas is always difficult to decide. From one angle, I prefer to think `schema.common` is the place for schemas more decoupled from domains. Something where we would describe a big integer, a public key, a theme, a style map, and so on. But an account schema could live outside `schema.common`. This would help us separate things based on bounded contexts. Otherwise there's a tendency `common` stuff can absorb anything.

An account can mean other things besides a wallet account. Maybe if we want to define a schema in the global registry, it should be in a different namespace, so the keyword in the registry would be `:schema.wallet/account`.

Or maybe we shouldn't couple quo components with domain concepts, even though they look like they depend on them. This would lead to a bit of duplication of schemas, but then we would be able to evolve schemas independently from quo. Just a tradeoff.

Long train of thought, just ideas on the table @shivekkhurana.

Reply:
There are two schools of typings that I have observed:
1. Move all types to their own ns
This helps avoid circular deps. that may arise in future

2. Keep types close to implementation
Better docs, easier to implement as no extra files are needed, available at the point of execution.

I personally prefer 2 (ie local types), but there are some domain models that will be repeated. `Account` being one of them maybe?

So in my head, I assumed `schema.common` to absorb any type that appears more than once.